src/view.coffee

```diff
@@ -1642,12 +1646,38 @@ define ['droplet-helper', 'droplet-draw', 'droplet-model'], (helper, draw, model
 
         # Blocks have a shape including a lego nubby "tab", and so
         # they need to be at least wide enough for tabWidth+tabOffset.
+
+        @extraWidth = 0
+        if 'add-button' in @model.classes
+          @extraWidth += @view.opts.buttonWidth + @view.opts.buttonPadding
+
+        if 'subtract-button' in @model.classes
+          @extraWidth += @view.opts.buttonWidth + @view.opts.buttonPadding
+
         for size, i in @minDimensions
           size.width = Math.max size.width,
               @view.opts.tabWidth + @view.opts.tabOffset
+          size.width += @extraWidth
 
         return null
 
+      drawSelf: (ctx, style) ->
+        super
+
+        start = @totalBounds.x + @totalBounds.width - @extraWidth
+
+        ctx.textBaseline = 'top'
+        ctx.font = @view.opts.textHeight + 'px '
+        ctx.fillStyle = '#000'
+
+        if 'add-button' in @model.classes
+          @addButtonRect.stroke(ctx, '#000')
```

Comment:
instead of stroking the rect directly, try getting a path (via rect.toPath) and then using the path's beveling feature to draw a beveled path.  Can you try it to see how it looks?